documentation/developer-guide/modules/tooling-guide/pages/bamm-cli.adoc

```diff
@@ -140,6 +140,214 @@ The full command would result in:
 java -jar bamm-cli-{sds-sdk-version}.jar aspect _AspectModel.ttl_ to openapi -b "https://www.example.org" -r "/resources/\{resourceId}" -p _fileLocation_
 ----
 
+=== Mapping between the Aspect models and the OpenAPI Specification
+
+In this section, a detailed description of the mapping between individual Aspect elements and the OpenAPI specification is given.
+To make it easier to follow, the mapping is explained based on a concrete example, divided into logically coherent blocks. Please bear in mind
+that these blocks are snippets or fragments of a larger whole; viewed in isolation they do not necessarily form a valid or meaningful aspect model or OpenAPI specification.
+
+==== Naming and versioning
+
+Please consider the following model fragment, with the attention focused on the numbered elements:
+
+----
+@prefix bamm: <urn:bamm:io.openmanufacturing:meta-model:2.0.0#>.
+@prefix : <urn:bamm:test:2.0.0#>. <1>
+
+:Test a bamm:Aspect;
+    bamm:name "Test"; <3>
+    bamm:preferredName "TestAspect"@en ; <2>
+    bamm:preferredName "TestAspekt"@de .
+----
+<1> full URN of :Test Aspect
```

Comment:
This is technically not correct, as the full URN includes also the Aspect name. Maybe something like: _<1> prefix used to build the full URN of :Test Aspect_ ?
Or does that sound unnecessarily complicated?